testData/solutions/win32.json5

```diff
@@ -5807,6 +6472,458 @@
+                    "getAction": "SPI_GETMOUSEBUTTONSWAP",
+                    "setAction": "SPI_SETMOUSEBUTTONSWAP",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "BOOL"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "SwapMouseButtonsConfig": {
+                        "schema": {
+                            "title": "Swap mouse primary button",
+                            "description": "Swap mouse left/right buttons",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    }
+                },
+                "capabilitiesTransformations": {
+                    "SwapMouseButtonsConfig": {
+                        "transform": {
+                            "type": "fluid.transforms.value",
+                            "inputPath": "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons",
+                            "outputPath": "value"
+                        },
+                        "path": {
+                            "transform": {
+                                "type": "fluid.transforms.literalValue",
+                                "input": {
+                                    "get": "pvParam",
+                                    "set": "uiParam"
+                                }
+                            }
+                        }
+                    }
+                },
+                "inverseCapabilitiesTransformations": {
+                    "http://registry\\.gpii\\.net/applications/com\\.microsoft\\.windows\\.mouseSettings.SwapMouseButtons": "SwapMouseButtonsConfig.value"
+                }
+            },
+            "configureScrollWheelMode": {
+                "type": "gpii.windows.spiSettingsHandler",
+                "liveness": "live",
+                "options": {
+                    "getAction": "SPI_GETWHEELSCROLLLINES",
+                    "setAction": "SPI_SETWHEELSCROLLLINES",
+                    "uiParam": 0,
+                    "pvParam": {
+                        "type": "UINT"
+                    },
+                    "fWinIni": "SPIF_UPDATEINIFILE|SPIF_SENDCHANGE",
+                    "verifySettings": true
+                },
+                "supportedSettings": {
+                    "ScrollWheelModeConfig": {
+                        "schema": {
+                            "title": "Mouse scroll wheel mode",
+                            "description": "Changes the number of lines to scroll when moving the scrollwheel.",
+                            "type": "integer",
+                            "default": 3,
+                            "minimum": 1,
+                            "oneOf": [
```

Comment:
@the-t-in-rtf - is it permissible to nest "maximum" within "oneOf" in this way? Please advise on improved expression of this structure

Reply:
As I understand it, the "type": "integer" itself would need to go inside the "oneOf", with the enum in the other branch: https://cswr.github.io/JsonSchema/spec/generic_keywords/

Reply:
They are overlaid on one another, i.e. you must satisfy all of the rules at the same level as the "oneOf", and one set of rules in the "oneOf" array.   So, the input here must be an integer, must be at least 1, and must either be less than or equal to 100, or must be the exact single enum value.  Of course,  it doesn't hurt anything to move the "type" and "minimum" into the same block as the "maximum", as that information is not all that useful within the "enum" path in the "oneOf".  In fact,  it makes the contract more readable, so I'd say, do it for that reason.

Also for readability purposes, I'd suggest adding [a "hint" element](https://github.com/the-t-in-rtf/gpii-json-schema/blob/GPII-3083/docs/gss.md#the-enumlabels-errors-and-hint-attributes) for at least the "maximum" branch of the "oneOf" array, so that the intent is clearer to someone working with a debugger, and so that we have the option to present guidance to end users.

Reply:
@the-t-in-rtf Sorry, I don't know how to follow with the hint element because I think we don't have a place for the needed message bundle. Should we talk about where to put one, before we proceed to create hints?

Reply:
@JavierJF, it's just like anything else, an English string, it'll get moved out in a later effort.  However, we talked to @sgithens, he's just using titles and descriptions at the moment, so you could also use a `title` instead of a hint (again, with the literal English wording for now).

Reply:
Thanks @the-t-in-rtf I will follow with that recommendation.

Reply:
Yes, this layout makes sense to me now, thanks